pkg/asset/installconfig/ovirt/validation.go

```diff
@@ -125,3 +126,22 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+// ValidateProvisioning - Perform Provisioning Validations for oVirt
+func ValidateProvisioning(ic *types.InstallConfig) error {
+	clusterName := ic.ObjectMeta.Name
+	zoneName := ic.BaseDomain
+	validFqdns := []string{
+		fmt.Sprintf("api.%s.%s", clusterName, zoneName),
+		fmt.Sprintf("test.apps.%s.%s.com", clusterName, zoneName),
```

Comment:
.com?

Reply:
yeap sorry , i've added for my local testing and didn't remove